WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/EditorPage.java

```diff
@@ -5,52 +5,99 @@
 
 import org.wordpress.android.R;
 
+import static android.support.test.InstrumentationRegistry.getInstrumentation;
 import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.Espresso.openActionBarOverflowOrOptionsMenu;
+import static android.support.test.espresso.Espresso.pressBack;
 import static android.support.test.espresso.action.ViewActions.click;
 import static android.support.test.espresso.action.ViewActions.typeText;
 import static android.support.test.espresso.assertion.ViewAssertions.matches;
 import static android.support.test.espresso.matcher.RootMatchers.isDialog;
-import static android.support.test.espresso.matcher.ViewMatchers.isDescendantOfA;
 import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
 import static android.support.test.espresso.matcher.ViewMatchers.withHint;
 import static android.support.test.espresso.matcher.ViewMatchers.withId;
 import static android.support.test.espresso.matcher.ViewMatchers.withText;
 import static org.hamcrest.Matchers.allOf;
+import static org.wordpress.android.support.WPSupportUtils.clickOn;
 import static org.wordpress.android.support.WPSupportUtils.isElementDisplayed;
+import static org.wordpress.android.support.WPSupportUtils.populateTextField;
 import static org.wordpress.android.support.WPSupportUtils.waitForElementToBeDisplayed;
+import static org.wordpress.android.support.WPSupportUtils.withIndex;
 
 public class EditorPage {
     private static ViewInteraction publishButton = onView(withId(R.id.menu_save_post));
-
     private static ViewInteraction editor = onView(withId(R.id.aztec));
-
-    private static ViewInteraction titleField = onView(allOf(withId(R.id.title), withHint("Title")));
-
-    private static ViewInteraction publishConfirmation = onView(allOf(
-            withText("Post published"), isDescendantOfA(withId(R.id.snackbar))));
+    private static ViewInteraction titleField = onView(allOf(withId(R.id.title),
+            withHint("Title")));
+    private static ViewInteraction publishConfirmation = onView(withText("Post published"));
```

Comment:
I think you may have changed this accidentally when merging in the mocks changes. For this to work reliably, I found this to be a good solution:

```suggestion
    private static ViewInteraction publishConfirmation = onView(withText("Post published"));
    private static ViewInteraction publishConfirmation = onView(allOf(
            withText("Post published"), isDescendantOfA(withId(R.id.snackbar))));
```